clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Command for downloading copybooks allows selections of data sets which should be downloaded
 - Code actions for an unknown operation code
 - Quick fixes for typos in instruction and macro names added to the code actions
+- Endevor and CICS preprocessor statements highlighting and parsing
```

Comment:
```suggestion
- CICS preprocessor statements highlighting and parsing
```
Endevor prep already has its changelog entry?

Reply:
I wanted to just append the previous entry with the CICS mention. Do you prefer a separate lines for Endevor and CICS?